config/codecov.map

```diff
@@ -16,7 +16,7 @@
 ^/(.*)/(.*)/(.*)/branch/(?P<branch>.+)/(graph|graphs)/badge.svg be_api
 ^/.*/.*/.*/(graph|graphs)/badge.svg be_api
 ^/login/(gh|github|ghe|github_enterprise|bb|bitbucket|bbs|bitbucket_server|gl|gitlab|gle|gitlab_enterprise|okta|sentry) be_api
-^/logout/(gh|github|ghe|github_enterprise|bb|bitbucket|bbs|bitbucket_server|gl|gitlab|gle|gitlab_enterprise|okta|sentry) be_api
+^/logout((\/)(gh|github|ghe|github_enterprise|bb|bitbucket|bbs|bitbucket_server|gl|gitlab|gle|gitlab_enterprise|okta|sentry)?)? be_api
```

Comment:
Sorry last thing, perhaps you don't need the `(\/)` and can just do `/`

```
^/logout(/(gh|github|ghe|github_enterprise|bb|bitbucket|bbs|bitbucket_server|gl|gitlab|gle|gitlab_enterprise|okta|sentry)?)?
```

Reply:
I wasn't sure exactly how the syntax here works, sometimes it's escaped sometimes not - will test it out rn!